RevenueCatUI/CustomerCenter/Views/UIKit Compatibility/CustomerCenterViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterViewController.swift
+//
+//  Created by Will Taylor on 12/6/24.
+
+import RevenueCat
+import SwiftUI
+
+#if canImport(UIKit) && os(iOS)
+
+/// Warning: This is currently in beta and subject to change.
```

Comment:
🤔 let's not even have this. We really shouldn't be changing the UI, and we're going GA very soon in any case.